doc/install/install-kubernetes.md

```diff
@@ -0,0 +1,120 @@
+# Install Kuadrant on a Kubernetes cluster
+
+> [!NOTE]  
+> You must perform these steps on each Kubernetes cluster where you want to use Kuadrant.
+
+
+## Prerequisites
+
+- Access to a Kubernetes cluster, with `kubeadmin` or an account with similar permissions
+
+## Procedure
+
+This guide will show you how to install Kuadrant onto a bare Kubernetes cluster.
+
+Alternatively, if you are looking instead for a way to set up Kuadrant locally to evaluate or develop, consider running the kind & Kubernetes [quickstart script](https://docs.kuadrant.io/latest/getting-started-single-cluster/).
+
+### Install Gateway API
+
+```bash
+kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.1.0/standard-install.yaml
+```
+
+### Install [OLM](https://olm.operatorframework.io/)
+
+```bash
+curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.23.1/install.sh | bash -s v0.23.1
+```
+
+### Install Istio as a Gateway API provider
+
+> [!NOTE]  
+> There are several ways to install Istio (via `istioctl`, Helm chart or Operator) - this is just an example for starting from a bare Kubernetes cluster.
+
+
+```bash
+curl -sSL https://istio.io/downloadIstio | ISTIO_VERSION=1.20.7 sh -
+./istio-1.20.7/bin/istioctl install --set profile=minimal -y
+./istio-1.20.7/bin/istioctl operator init
+kubectl apply -f https://raw.githubusercontent.com/Kuadrant/kuadrant-operator/main/config/dependencies/istio/istio-operator.yaml
+```
+
+### Install Kuadrant
+
+```bash
+kubectl create -f https://operatorhub.io/install/kuadrant-operator.yaml
+```
+
+### Request a Kuadrant instance
+
+```bash
+kubectl create namespace kuadrant-system
+kubectl -n kuadrant-system apply -f - <<EOF
+apiVersion: kuadrant.io/v1beta1
+kind: Kuadrant
+metadata:
+  name: kuadrant
+spec: {}
+EOF
+```
+
+Kuadrant should now install. You can check the operator's install status with:
+
+```bash
+kubectl wait --for=jsonpath='{.status.state}'=AtLatestKnown subscription/my-kuadrant-operator -n operators --timeout=600s
+```
+
+Kuadrant is now ready to use.
```

Comment:
This split between namespaces look expected to you?

```shell
$ kubectl get po -n kuadrant-system
NAME                                   READY   STATUS    RESTARTS   AGE
authorino-8584b6d457-qp6pg             1/1     Running   0          101s
limitador-limitador-78b6b5fbc9-mv2xp   1/1     Running   0          101s

$ kubectl get po -n operators
NAME                                                     READY   STATUS    RESTARTS        AGE
authorino-operator-7765dd74c-r6tjf                       1/1     Running   1 (2m56s ago)   3m39s
authorino-webhooks-bcc654989-q57nl                       1/1     Running   1 (2m57s ago)   3m39s
cert-manager-597bfdf55f-wj25v                            1/1     Running   0               3m13s
cert-manager-cainjector-74d4cb6c5f-j4ms6                 1/1     Running   3 (2m19s ago)   3m13s
cert-manager-webhook-657bd6f755-trnnm                    1/1     Running   3 (2m19s ago)   3m12s
dns-operator-controller-manager-94bf9d9dd-gd75w          1/1     Running   1 (2m30s ago)   3m15s
kuadrant-operator-controller-manager-6b784bd95d-d6nkd    1/1     Running   1 (2m57s ago)   3m40s
limitador-operator-controller-manager-78c4c87568-qr267   1/1     Running   1 (2m55s ago)   3m45s
```

Reply:
this is OK